tests/data_manager/test_prices_manager.py

```diff
@@ -34,21 +36,20 @@ async def test_initialize(backtesting_exchange_manager):
     prices_manager.mark_price_set_time = 10
     prices_manager.valid_price_received_event.set()
 
-    await prices_manager.initialize_impl()
+    await prices_manager.initialize()
     assert prices_manager.mark_price == prices_manager.mark_price_set_time == 0
     assert not prices_manager.valid_price_received_event.is_set()
 
 
-async def test_set_mark_price(backtesting_exchange_manager):
-    prices_manager = PricesManager(backtesting_exchange_manager)
+async def test_set_mark_price(prices_manager):
     prices_manager.set_mark_price(10)
     assert prices_manager.mark_price == 10
-    assert prices_manager.mark_price_set_time == backtesting_exchange_manager.exchange.get_exchange_current_time()
-    assert prices_manager.valid_price_received_event.is_set()
+    if not os.getenv('CYTHON_IGNORE'):
```

Comment:
why is it not working in cython anymore ?

Reply:
I don't know how it worked but `prices_manager.exchange_manager` can't be accessed when testing with cython compilation.